matchengine/plugins/DFCITrialMatchDocumentCreator.py

```diff
@@ -34,7 +34,7 @@ def get_genomic_details(genomic_doc: Dict, trial_match: TrialMatch):
         alteration.append(hugo_symbol)
 
     # add mutation
-    if true_protein is not None:
+    if true_protein is not None and true_protein:
```

Comment:
Good catch. @jasonhansel do we already have that fixed or 😬

Reply:
We should be able to incorporate this change without issues.